tests/integration/interface_common_test.py

```diff
@@ -16,11 +16,21 @@
 # You should have received a copy of the GNU Lesser General Public License
 # along with this program. If not, see <https://www.gnu.org/licenses/>.
 #
+import pytest
```

Comment:
nit: pytest is a 3rd party package, please place below time with a newline in between.

Reply:
Done